GX-TimeFrequency/tests/test_minimax/refs/32_freq_time_points.dat

```diff
@@ -0,0 +1,34 @@
+ Minimax Grid
```

Comment:
Consider moving  GX-TimeFrequency/tests/test_minimax/output_32_freq_time_points.dat  to this file, such that the formatting is consistent